soyuz_app/tests/test_views.py

```diff
@@ -152,6 +152,19 @@ def test_signup_max_capacity_POST_add_to_waiting_list(self):
         self.assertEquals(self.waiting_list1.users.all().count(), 3)
         self.assertTemplateUsed(response, 'users/waiting-list-confirmation.html')
 
+    def test_signup_new_user_GET(self):
```

Comment:
Why is GET capitalised? Why are we signalling GET request when the request in the test is a POST request?